napps/ofcore/main.py

```diff
@@ -81,7 +81,7 @@ def handle_raw_message_in(self, event):
         self.add_to_msg_in_buffer(new_event)
 
     @ListenTo('KycoMessageInHello')
```

Comment:
Yes, I see. But since that the develop branch has already the changes, shouldn't you do a rebase first ? If I accept this PR, this will brake the develop,right ?